vmss_extension/03-update-instances.yml

```diff
@@ -0,0 +1,27 @@
+# Description
+# ===========
+# This playbook updates Virtual Machine Scaleset instances.
```

Comment:
maybe clear information on why need upgrades the vmss instances. it's "upgrade" instead of "update""

Reply:
so why need upgrade here? it's requrired after install custom script extension?

Reply:
no, existing instances need "upgrade" only new instances will be automatically flashed with new image + extensions. mechanism is exactly the same as changing image.